src/main/java/taxi/util/Encrypt.java

```diff
@@ -0,0 +1,26 @@
+package taxi.util;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.KeySpec;
+import java.util.Base64;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+
+public class Encrypt {
```

Comment:
I'd recommend not using password hashing for this task. You will do it in the future using the best approaches